kitchen-tests/Berksfile

```diff
@@ -2,3 +2,4 @@ source "https://supermarket.chef.io"
 
 cookbook "end_to_end", path: "cookbooks/end_to_end"
 cookbook "rspec", path: "cookbooks/rspec"
+cookbook "chef-client", git: "https://github.com/tyler-ball/chef-client.git", branch: "chef_license_accept"
```

Comment:
that should probably get updated

Reply:
I don't have access to that repo to push a branch. If someone gives me access I can change that, otherwise this needs to stay

Reply:
Or we need to finish the changes to that cookbook and get the merged before this PR merges

Reply:
... Thinking about it, we should release the `chef-client` cookbook changes before releasing this, even if its only going to the `current` channel on PR merge